apps/dashboard/app/javascript/stylesheets/pinned_apps.scss

```diff
@@ -0,0 +1,74 @@
+$launcher_border: rgb(45,155,240);
+$launcher_border_hover: rgb(0,95,133);
+
+.app-launcher-disabled::before {
+  content: '';
+  position: absolute;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  background-color: rgba(255, 255, 255, 0.6);
+}
+
+.app-launcher-spinner {
+  position: absolute;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  background-color: rgba(0, 0, 0, 0.5);
+  color: #fff;
+
+  .spinner-border {
+    width: 60px;
+    height: 60px;
+  }
+}
+
+.app-launcher {
+  position: relative;
+  height: 95%;
+  margin-bottom: 20px;
+
+  border-radius: 30px;
+  box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.5);
```

Comment:
While shadow may be ok to use px values, can we use `em` for `margin-bottom` and percentages for `border-radius`?